### DISCUSSION TITLE
Commercial license for Brakeman - contact info

### DISCUSSION THREAD

**Discussion by mlinderman** [presidentbeef/brakeman#1731] [Q&A] [answered]
Hello, apologies for writing about this topic here but it's been unproductive to seek help from Synopsys who I understand acquired the Brakeman product.  My company is interested in understanding whether a commercial license is available for Brakeman.  Can you connect me with someone who can provide information about that?  Thanks.

**Comment by presidentbeef** [comment, marked as answer]
A commercial license is only required if you are re-selling Brakeman in some fashion (e.g. as part of a SaaS product).

I have no idea if Synopsys is still willing to work with companies that need a commercial license. If you need one (again, actually selling Brakeman as part of a service or product) please contact me at justin@brakemanpro.com and I'll send you the contact information I have.
